src/js/ripple/serializedobject.js

```diff
@@ -115,9 +119,9 @@ SerializedObject.check_no_missing_fields = function(typedef, obj) {
   if (missing_fields.length > 0) {
     var object_name;
 
-    if (obj.TransactionType !== void(0)) {
+    if (obj.TransactionType !== undefined) {
       object_name = SerializedObject.lookup_type_tx(obj.TransactionType);
-    } else if (obj.LedgerEntryType != null){
+    } else if (obj.LedgerEntryType !== undefined) {
```

Comment:
I always use != null, cause I learned js from looking at compiled coffee code. iirc existence operator `?` uses `!= null`. But the linter was crying. Using `if ( bla )` probably maintain semantics better, but it's not a big deal here eh?